### DISCUSSION TITLE
General Question

### DISCUSSION THREAD

**Discussion by JayDee56** [apprenticeharper/DeDRM_tools#2085] [Q&A] [answered]
I've been using Calibre for multiple years now.  A while back when Amazon switched to the AZW files and there were problems stripping the DRM I rolled back to version 4.23 and use the kindle "false" command when opening Kindle for desktop.  It works.  I just got the DeDRM tools 7.2.1 but it's telling me I need a newer version of Calibre to install them.  If I update Calibre will I continue to be able to import books purchased at Amazon or should I just continue using the older version?  Thanks for any help.

**Comment by ElleKayEm** [comment, marked as answer]
Yes, you should be good.  There's actually an even newer version of DeDRM here:  https://github.com/noDRM/DeDRM_tools/releases/tag/v10.0.2

**Reply by JayDee56** [reply]
Thank you for the quick reply.
